pyprep/prep_pipeline.py

```diff
@@ -54,16 +54,38 @@ class PrepPipeline:
     def __init__(self, raw, prep_params, montage, ransac=True, random_state=None):
         """Initialize PREP class."""
         self.raw = raw.copy()
-        self.ch_names = self.raw.ch_names
+
+        # separe eeg and non eeg channels
+        self.raw_non_eeg = self.raw.copy()
+        self.chs_all = self.raw.ch_names.copy()
+        self.chs_types = self.raw.get_channel_types()
```

Comment:
These are *all* types, so I would find `ch_types_all` more straight forward to understand

also, for all of these variables, I would find `ch_...` instead of `chs_...` better. Speaking aloud it would be "channels types" versus "channel types ... and the latter makes more sense IMHO.

Note: I make one suggestion here to clarify my point, but adhering to this point would mean many more small changes in code to adjust all the variable and attribute names.
```suggestion
        self.ch_types_all = self.raw.get_channel_types()
```

Reply:
I agree, to be honest I was sticking to the 'chs' way because the input parameters are 'ref_chs' and 'reref_chs' but in a way those are different grammar contexts lol